ui/cryptomaterial/dropdown.go

```diff
@@ -193,11 +203,14 @@ func (d *DropDown) layoutOption(gtx layout.Context, itemIndex int) D {
 				Right: unit.Dp(5),
 				Left:  unit.Dp(5),
 			}.Layout(gtx, func(gtx C) D {
-				lbl := d.theme.Body2(item.Text)
-				if !d.isOpen && len(item.Text) > 14 {
-					lbl.Text = item.Text[:14] + "..."
+				txt := item.Text
+				if !d.isOpen && len(txt) > 14 {
+					txt = item.Text[:14] + "..."
+				} else if txt == "" {
+					txt = "Select a wallet"
```

Comment:
No. You can pass a `prompt string` but don't hard code this for all dropdowns that have no txt.